specification/azurearcdata/resource-manager/Microsoft.AzureArcData/preview/2022-03-01-preview/sqlServerInstances.json

```diff
@@ -119,6 +119,8 @@
           "type": "string",
           "description": "SQL Server license type.",
           "enum": [
+            "LicenseOnly",
+            "ServerCal",
```

Comment:
the reason to add this 2 new value is match your service implementation. or it is new feature? and can it add into end of enum or before Undefined?

Reply:
Match to new implementation. It is a string. Does the sequence matter? If it is enum, undefined should be the first. Not sure why they use the reverse order.

Reply:
I added "Payg" last Friday. PM asks us to change to "PAYG". So I changed it in this PR. We have no code to use the new value anywhere so far.